acceptance-tests/tests/build.gradle

```diff
@@ -12,10 +12,19 @@
  */
 
 plugins {
-  id 'org.web3j' version '4.8.9'
+  id 'org.web3j' version '4.9.2'
   id 'org.web3j.solidity' version '0.3.5'
 }
 
+configurations.all {
+  resolutionStrategy.eachDependency { DependencyResolveDetails details ->
+    if (details.requested.group == 'org.web3j' && details.requested.version == '4.9.2') {
+      details.useVersion '4.9.4'
+      details.because 'web3-gradle-plugin has not been updated yet'
+    }
+  }
+}
+
```

Comment:
Should be solved in 

https://github.com/hyperledger/besu/pull/3752 

don't mind which approach is taken.

Reply:
Me neither, maybe it makes more sense to merge that other PR first since it covers specifically the web3j upgrade, but I do not mind if the approach currently implemented in my branch is finally taken.

cc: @diega

Reply:
I like this one! @freemanzMrojo would you push it into #3752? I think it's better to keep the PRs/squashes split if possible

Reply:
I can do that @diega , thanks!

Reply:
Will include @shemnon wording suggestions as well @diega

Reply:
Could you give me permission to your `besu` fork @diega  so I can push my commit? Thanks in advance!

Reply:
sure! check now

Reply:
Just pushed the commit, thanks!